docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,62 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
+
+For the first run, you need these commands:
+
+```bash
+./prepare-workspace.sh
+./ci/build-test.sh
+```
+
+If you have modified anything in the code generator or docs linter tool;
+ to re-test these you need to re-run this command:
+
+```bash
+./ci/build-test.sh
+```
+
+If you have modified anything in the Unity Project; to re-test this you can
+ re-run this command:
+
+```bash
+./ci/test.sh
+```
+
+## Unity tests
+
+For Unity tests, you can also run tests from the Unity Test Runner Window.
```

Comment:
(1) Am I testing Unity or my project or the GDK? What do we mean by "Unity tests"?
(2) Is "window" part of the name of the window (IE it's written in the window itself) or is it a description of what's on screen? If it's a description of what's on screen, it's lower case 'w'. That is, "Unity Test Runner window". 
(3) Does the "Test Runner" window have capital initial letters for "Test Runner"? We need to make sure the text matches what's on screen because this gives the user confidence in the docs and also helps orient them in the GUI.